README.md

```diff
@@ -31,7 +31,7 @@ Some API methods have optional parameters that can be passed. For example:
 client := github.NewClient(nil)
 
 // list recently updated repositories for org "github"
-opt := &github.RepositoryListByOrgOptions{Sort: "updated"}
+opt := &github.RepositoryListByOrgOptions{Type: "member"}
```

Comment:
I honestly don't remember what "member" means for an organization repo.  Maybe change this to "public", and then update the comment above to reflect that.  Same in doc.go